components/events/eventCard.tsx

```diff
@@ -2,43 +2,85 @@ import event from "@/backend/schemas/event";
 import Image from "next/image";
 import Link from "next/link";
 
-// Interface for the event card
-interface EventProps{
-    imageUrl: string,
-    title: string,
-    date?: string,
-    time?: string,
-    description: string,
-    slug: string
-} 
+// Define the props interface for the EventCard component
+interface EventProps {
+  imageUrl: string;
+  title: string;
+  date?: string;
+  time?: string;
+  description: string;
+  slug: string;
+}
 
-// Event card itself
-export default function EventCard({date, title, imageUrl, time, description, slug}: EventProps): JSX.Element {
-    
-    return (
-        <div className="flex flex-col items-center border rounded-lg shadow md:flex-row md:max-w-xl max-w-md border-gray-700 bg-gray-800 hover:bg-gray-700">
-            <Image
-                className="object-cover w-full rounded-t-lg h-96 md:h-48 md:w-48 md:rounded-none md:rounded-l-lg"
-                src={imageUrl}
-                alt="project card"
-                width={400}
-                height={400}
-            />
-            <div className="flex flex-col justify-between p-4 leading-normal"> 
-                <h5 className="mb-2 text-2xl font-bold tracking-tight text-white">
-                    {title}
-                </h5>
-                <p className="mb-3 font-normal text-gray-400">
-                    {description}
-                </p>
-                <h4> 📅 {date}</h4>
-                <h4>🕕 {time}</h4>
-                <Link href={`/arrangementer/${slug}`} className="inline-flex items-center px-3 py-2 mt-2 text-sm font-medium text-center text-white  rounded-lg focus:ring-4 focus:outline-none  bg-blue-600 hover:bg-blue-700 focus:ring-blue-800">
-                    Les meir
-                    <svg aria-hidden="true" className="w-4 h-4 ml-2 -mr-1" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fillRule="evenodd" d="M10.293 3.293a1 1 0 011.414 0l6 6a1 1 0 010 1.414l-6 6a1 1 0 01-1.414-1.414L14.586 11H3a1 1 0 110-2h11.586l-4.293-4.293a1 1 0 010-1.414z" clipRule="evenodd"></path></svg>
-                </Link>
-            </div>
+// Define the EventCard component
+const EventCard: React.FC<EventProps> = ({
+  date,
+  title,
+  imageUrl,
+  time,
+  description,
+  slug,
+}: EventProps) => {
+  // Calculate the event date and time as a JavaScript Date object
+  const eventDateTime = date && time ? new Date(`${date}T${time}`) : null;
```

Comment:
Looks good, but I cant make it work locally.

Reply:
<img width="813" alt="consolelog" src="https://github.com/IT-Start-Gjovik/startgjovik_website/assets/90150033/47cd109a-f5c9-43c1-9574-f29972976b4a">
![consoleoutput](https://github.com/IT-Start-Gjovik/startgjovik_website/assets/90150033/6ff73653-2750-4718-844e-ffb40064de8f)
![output](https://github.com/IT-Start-Gjovik/startgjovik_website/assets/90150033/8b7c09ef-3b72-4b5f-88eb-e4b465f1782d)

Reply:
![consoleoutput](https://github.com/IT-Start-Gjovik/startgjovik_website/assets/90150033/5273fa7f-9120-450b-9d77-ce251b6fe1e4)

Reply:
![Uploading output.png…]()

Reply:
I'm not quite sure why it won't work for you, but it could be something with the date and time. The values may not be in the expected format (ISO 8601 format)?